lib/src/widgets/custom_button.dart

```diff
@@ -0,0 +1,122 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+
+import '../theme/theme.dart';
+import 'translated_text.dart';
+
+enum CustomButtonStyle {
+  fancy,
+  outlined,
+  filled,
+}
+
+class CustomButton extends StatelessWidget {
```

Comment:
I am not really a favour of using 'custom' in class names. I always wonder what is custom with respect to what. Maybe we can stick to the naming of the previous class, which was `IrmaThemedButton`, so it can be `YiviThemedButton` now.